modules/end-user-guide/partials/proc_mounting-a-secret-as-a-file-into-a-workspace-container.adoc

```diff
@@ -9,21 +9,82 @@ WARNING: On
 ifeval::["{project-context}" == "che"]
 {platforms-name} older than v1.13 and
 endif::[]
-OpenShift {ocp3-ver}, secrets mounted as file overrides volume mounts defined in the devfile.
+OpenShift {ocp3-ver}, secrets mounted as a file override volume mounts defined in the devfile.
 
-This section describes how to mount a secret from the user's {orch-namespace} as a file in single-workspace or multiple-workspace containers of {prod-short}.
+A secret can be mounted from the user's {orch-namespace} as a file in single-workspace or multiple-workspace containers of {prod-short}.
 
 .Prerequisites
 
-* A running instance of {prod-short}. To install an instance of {prod-short}, see {link-installing-an-instance}.
+* A running instance of {prod-short}.
++
+To install an instance of {prod-short}, see {link-installing-an-instance}.
 
 .Procedure
 
-. Create a new {platforms-name} secret in the {platforms-name} {orch-namespace} where a {prod-short} workspace will be created.
+To mount a secret as a file, complete the following steps as shown in the next example:
+
+. Create a new {platforms-name} secret in the {platforms-namespace} of the {prod-short} workspace.
+. Match the secret's `labels` to the set of labels configured in the `che.workspace.provision.secret.labels` property of {prod-short}.
```

Comment:
Well, it doesn't seem to be the case that the `labels` are in the secret (these labels identify the secret wherever they appear), so "Match the `labels` in the secret" is likely not going to work here. "Match the `labels` for the secret ..." could work, but this would make it harder to follow the whole sentence ("Match the `labels` for the secret to the set of labels configured in the ....")